python/spider-py/pyproject.toml

```
@@ -1,13 +1,19 @@
 [project]
 name = "spider-py"
-version = "0.0.1"
+version = "0.0.0-beta.0"
```

```suggestion
version = "0.0.1-beta.0"
```
I think we should still use `0.0.1` since `0.0.0` isn't a valid SemVer.